webapp/channels/src/components/integrations/outgoing_oauth_connections/oauth_connection_audience_input.test.tsx

```diff
@@ -89,89 +89,90 @@ describe('components/integrations/outgoing_oauth_connections/OAuthConnectionAudi
         const props = {...baseProps};
         const state = stateFromOAuthConnections({});
         const store = mockStore(state);
-        await act(async () => {
```

Comment:
i never ~~liked~~ understood act

Reply:
I want to write some docs on that now that I kind of understand it.

My short explanation for it is that you need to run `act` whenever something happened that React needs to respond (like handling events, processing `setState` calls, or handling async functions). You don't need it when RTL automatically uses it though (like when using `userEvent` actions)